pagopa/pn-frontend
# Contacts page not accessible if user is an operator - pn-5323

## Short description
If a PG user is an operator, he can't navigate to contacts page

## List of changes proposed in this pull request
- Added hasUserPermissions hook in pn-commons
- Added PrivateRoute component in pn-commons
- Changed routes.tsx to manage user permissions
- Changed App.tsx to prevent contacts api calls if user doesn't have permissions
- Added DomicileBanner component
- Changed Notifiche.page.tsx and NotificationDetail.page.tsx to show/hide DomicileBanner based on user permissions

## How to test
- Log with admin user and check that contacts page is available and banner is shown
- Log with operator user and check that contacts page is not accessible and banner is hidden
